index.html

```diff
@@ -9332,7 +9333,7 @@ <h5>Presentational Role Inheritance</h5>
 					</tr>
 					<tr>
 						<th class="role-childpresentational-head" scope="row">Children Presentational:</th>
-						<td class="role-childpresentational"> </td>
+						<td class="role-childpresentational"> </td>
```

Comment:
I'm not able to roll back this diff with a change suggestion. Not sure why, but it's not as simple as a different whitespace char.

Reply:
@chlane do you think you can remove this change? other wise this looks good to go!

Reply:
@spectranaut not sure why this whitespace issue is happening. I couldn't tell what to change on my end but the commit button was enabled after I edited line 9345. I just deleted and replaced the whitespace.